Directory.Build.targets

```diff
@@ -34,4 +41,29 @@
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsSourceProject)' == 'true'">
+    <AssemblyAttribute Include="System.CLSCompliantAttribute" Condition="'$(CLSCompliant)' == 'true'">
+      <_Parameter1>true</_Parameter1>
+    </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Reflection.AssemblyDefaultAliasAttribute">
+      <_Parameter1>$(AssemblyName)</_Parameter1>
+    </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Reflection.AssemblyDescriptionAttribute">
```

Comment:
I was checking to see if we explicitly set this in either runtime or corefx and couldn't find it.  I think it was being set through https://github.com/dotnet/sdk/blob/56f35a2e033b938a5e9d87da80ff2819a81e01bb/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets#L87-L89

I think that was set here: https://github.com/dotnet/runtime/blob/def5e3240bdee3ee37ba22c41c840bbf431c4b15/eng/versioning.targets#L7-L8

And here:
https://github.com/dotnet/corefx/blob/b86ce69c316b4b088356b89bab80f7fa79ff799c/eng/versioning.targets#L41-L42

Reply:
Thanks for sharing the links.

I see that in those linked cases the description was added via an additional `<Description>` property. I know you were not suggesting any changes, but I was wondering if there was a good reason for us to add that extra property here, and I am concluding it wouldn't be necessary. I think we can keep these lines as proposed in this PR.